internal/utils/config.go

```diff
@@ -343,6 +345,11 @@ type (
 		ApiKey           string          `toml:"-" mapstructure:"api_key"`
 	}
 
+	sessions struct {
+		Timebox           float64 `toml:"timebox"`
+		InactivityTimeout float64 `toml:"inactivity_timeout"`
```

Comment:
Please also update [init_config.toml](https://github.com/supabase/cli/blob/main/internal/utils/templates/init_config.toml) and init_config_test.toml with example values.